tests/Neo.UnitTests/UT_ProtocolSettings.cs

```diff
@@ -47,7 +47,7 @@ public void TestGetMemoryPoolMaxTransactions()
         [TestMethod]
         public void TestGetMillisecondsPerBlock()
         {
-            Assert.AreEqual((uint)15000, (uint)TestProtocolSettings.Default.MillisecondsPerBlock);
+            Assert.AreEqual((uint)3000, (uint)TestProtocolSettings.Default.MillisecondsPerBlock);
```

Comment:
It's OK to try 150000 because we will not use `MillisecondsPerBlock` from ProtocolSettings anymore.